lib/liquid/parse_context.rb

```diff
@@ -0,0 +1,39 @@
+module Liquid
+  class ParseContext
```

Comment:
We don't need to worry too much about API compatibility - breaking changes are already present in Liquid 4.